### PR TITLE
layout adjustment of event hanlder conditions and policy actions #2557

### PR DESCRIPTION
**Pull request by NNLemling**
slight change of the layout of the event handler conditions and simultaneous adjustment of the policy actions

### CODE REVIEW

File: privacyidea/static/components/config/views/config.policies.details.html [hunk 1]
```diff
@@ -270,9 +270,7 @@ <h3 translate>Condition</h3>
         <div class="tab-pane action" id="action">
             <h3 translate>Action</h3>
             <div class="form-group">
-                <label for="action" class="col-sm-3 control-label">
-                    <translate>Action</translate>
-                    <br>
+                <label for="action" class="col-sm-3">
```
Comment (cornelinux) [inline privacyidea/static/components/config/views/config.policies.details.html:273]: THe label in the event handlers has the class ``pl-0px``.

File: privacyidea/static/components/config/views/config.events.details.html [hunk 2]
```diff
@@ -133,7 +133,7 @@ <h3 translate>Condition</h3>
                         </button>
                     </label>
 
-                    <div class="input-group col-sm-9">
+                    <div class="input-group col-sm-12">
```
Comment (cornelinux) [inline privacyidea/static/components/config/views/config.events.details.html:136]: The columns are divided into 12 sections. So all ``col`` class should add up to 12.

col-sm (column small)...

So in line 123 you have col-sm-3. This should be accompanied by col-sm-9.
So why change this to col-sm-12. This looks wrong to me.

**Review by cornelinux — CHANGES_REQUESTED** [review]

File: privacyidea/static/components/config/views/config.policies.details.html (hunk 1 above)
Comment (NNLemling) [reply privacyidea/static/components/config/views/config.policies.details.html:273]: yes for some reason the "show selected conditions only" button is positioned further to the right than it should. the class pl-0px solves this problem. Strangely enough, that's not the case with the policy actions.

File: privacyidea/static/components/config/views/config.events.details.html (hunk 2 above)
Comment (NNLemling) [reply privacyidea/static/components/config/views/config.events.details.html:136]: Bildschirmfoto vom 2020-12-11 22-59-05

**Review by cornelinux — APPROVED** [review]

**Pull request merged**